content/cucumber/api.md

```diff
@@ -300,7 +300,13 @@ if (scenario.isFailed()) {
 ```
 
 ```javascript
-TODO: See Ruby or Java example
+if (scenario.isFailed()) {
+  webDriver.takeScreenshot().then(function(image,err) {
```

Comment:
Should return this promise, as shown in https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/step_definitions.md. https://github.com/cucumber/cucumber-js/blob/master/docs/support_files/hooks.md